src/server.c

```diff
@@ -5073,6 +5075,7 @@ int main(int argc, char **argv) {
     aeSetAfterSleepProc(server.el,afterSleep);
     aeMain(server.el);
     aeDeleteEventLoop(server.el);
+    freeErrorStrings();
```

Comment:
this is actually dead code.. the normal place where redis exists, it with a call to `exit` after calling `prepareForShutdown`. but anyway, redis doesn't bother to release memory or shutdown.. that's a waste of time. i suggest to remove this line.